client/src/main/java/com/arcbees/beeshop/client/application/home/HomeView.java

```diff
@@ -34,24 +44,131 @@
     PricePresenter shirtPrice;
     @UiField(provided = true)
     PricePresenter bagPrice;
+    @UiField(provided = true)
+    PricePresenter cupPrice;
+    @UiField(provided = true)
+    PricePresenter casePrice;
+    @UiField(provided = true)
+    PricePresenter mugPrice;
+    @UiField(provided = true)
+    PricePresenter keyPrice;
+    @UiField
+    AnchorElement gaeAnchor;
+    @UiField
+    AnchorElement jukitoAnchor;
+    @UiField
+    AnchorElement gwtpAnchor;
+    @UiField
+    AnchorElement arcbeesAnchor;
+    @UiField
+    AnchorElement gqueryAnchor;
+    @UiField
+    AnchorElement gsssAnchor;
+    @UiField
+    AnchorElement chosenAnchor;
```

Comment:
Do you have plans on abstracting the product registration logic? and also presenters should be created by the presenter and set into a slot.